README.md

```diff
@@ -56,12 +56,10 @@ dependencies {
    ...
    compile project(':RCTWeChat')    // Add this line only.
 }
-```
```

Comment:
@xing-zheng Why removing this? here is a little bit more wild, could you please format better?